ntc_templates/templates/cisco_ios_show_vlan_group.textfsm

```diff
@@ -0,0 +1,6 @@
+Value Required VLAN_GROUP_NAME (\S+)
+Value VLAN_ID (\d+.*)
```

Comment:
It would be preferable to make this a List, and call it VLAN_IDS

Reply:
> It would be preferable to make this a List, and call it VLAN_IDS

Jacob wondered if this output might wrap.

From my test, the vlan group list ends up on a single long line and does not have newlines injected.

```
switch#show vlan group
vlan group TEST : 2, 4, 6, 8, 10-15, 20, 22, 24, 26, 28, 30, 99-199, 201-299, 301-399, 401-499, 501-599, 601-699, 701-799, 801-899, 901-999, 1001-1010, 1012-1013, 1015-1016, 1018-1019, 1021-1022, 1024-1027, 1029-1032, 1034-1035
```

A `List` data type does not seem to be needed.
Certainly this leaves post-processing work to split the string into individual VLAN IDs, but that's how it does with TextFSM.